src/MICore/CommandFactories/gdb.cs

```diff
@@ -33,6 +33,11 @@ public override bool SupportsStopOnDynamicLibLoad()
             return true;
         }
 
+        public override bool SupportsChildProcessDebugging()
+        {
+            return true;
```

Comment:
Or does GDB support it there too?

---
In reply to: [59229655](https://github.com/Microsoft/MIEngine/pull/259#discussion_r59229655) [](ancestors = 59229655)